apps/benchmarks/benchmarks.nim

```diff
@@ -0,0 +1,44 @@
+import
+  math,
+  std/sequtils,
+  stew/results,
+  options,
+  ../../waku/waku_rln_relay/protocol_types,
+  ../../waku/waku_rln_relay/rln,
+  ../../waku/waku_rln_relay,
+  ../../waku/waku_rln_relay/conversion_utils,
+  ../../waku/waku_rln_relay/group_manager/static/group_manager
+
+import std/[times, os]
+
+let rlnIns = createRLNInstance(20).get()
+let credentials = toSeq(0 .. 1000).mapIt(membershipKeyGen(rlnIns).get())
+
+# Run as:
+# make benchmarks LOG_LEVEL=NOTICE
+# ./build/benchmarks
+let manager = StaticGroupManager(
+  rlnInstance: rlnIns,
+  groupSize: 1000,
+  membershipIndex: some(MembershipIndex(900)),
+  groupKeys: credentials,
+)
+
+manager.init()
+
+let data: seq[byte] = newSeq[byte](1024)
+
+var proofGenTimes: seq[times.Duration] = @[]
+var proofVerTimes: seq[times.Duration] = @[]
+
+for i in 0 .. 50:
+  var time = getTime()
```

Comment:
In case we want to measue CPU-bounded tasks, it would be interesting to use `cpuTime` instead.